fix(respect): referenced workflow execution flow inconsistency (#2966)
Redocly/redocly-cli: .changeset/child-workflow-failure-stops-parent.md

---
'@redocly/respect-core': patch
'@redocly/cli': patch
---

> Fixed `respect` to stop parent workflow execution when a step that references another workflow fails. Previously, the next steps of the parent workflow were still executed after the referenced workflow failed.

Reviewer: ```suggestion
Fixed an issue in `respect` where the execution of parent workflow's steps didn't halt after a step that referenced another workflow had failed.
```
